### PR TITLE
(fix) filepath on windows

### PR DESCRIPTION
**Pull request by itscaro**
To able to run test on windows

### CODE REVIEW

**Comment by mumoshu** [comment]
@itscaro Hey! Thanks for the PR. Would you mind helping me how I could verify this is working as intended? I have no windows machine on my stock

**Comment by itscaro** [comment]
Hi, maybe we could run some windows tests on circleci?

**Comment by itscaro** [comment]
Tried to add a simple job with windows on CircleCI but I got this error:
```
Creating a dedicated VM with windows-default image
failed to create host: Image windows-default is not supported
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -75,6 +75,20 @@ jobs:
       - run: make pristine
       - run: make test
 
+  test_windows:
+    machine:
+      image: windows-default
```
Comment (mumoshu) [inline .circleci/config.yml:80]:
```suggestion
    executor: win/default
```

I have literally no experience with it but reading CircleCI docs, the machine image isn't available in the Cloud version of CircleCI.

What if we used the `win/defaut` executor as documented?

https://circleci.com/docs/2.0/executor-types/#using-the-windows-executor

Comment (itscaro) [reply .circleci/config.yml]: let's try, I have no experience with CircleCI either :)

Comment (itscaro) [reply .circleci/config.yml]: @mumoshu it did not work, from what I searched, I think there is a toggle flag on Circle CI side to activate the ability to use windows runner

**Comment by itscaro** [comment]
There is no support for Windows as free plan for OSS:
https://circleci.com/open-source/

image

**Comment by mumoshu** [comment]
@itscaro Hey! Thanks for your efforts. Can we revert CI changes and merge only code changes for now? I think the code changes are valuable on its own.

**Comment by itscaro** [comment]
Hi @mumoshu I will separate in another PR for the change for CI.

**Comment by itscaro** [comment]
@mumoshu I separate the addition of windows build in CircleCI to https://github.com/itscaro/helmfile/tree/windows2
Since the build now works, it reveals more incompatibility in windows in test phrase, I will try to make tests work in windows build.

**Review by mumoshu — APPROVED** [review]
Thanks for your effort @itscaro!

**Pull request merged**